frontend/src/metabase/admin/datamodel/metadata/components/MetadataEditor/MetadataEditor.unit.spec.tsx

```diff
@@ -1,546 +0,0 @@
-
-      await userEvent.type(targetInput, "Products");
-      expect(popover.getByText("Products → ID")).toBeInTheDocument();
-    });
-
-    it("should show an access denied error if the foreign key field has an inaccessible target", async () => {
-      await setup();
-      await userEvent.click(screen.getByText(ORDERS_TABLE.display_name));
-
-      const section = within(
-        await screen.findByLabelText(ORDERS_USER_ID_FIELD.name),
-      );
-      expect(
-        section.getByPlaceholderText("Field access denied"),
-      ).toBeInTheDocument();
-    });
-
-    it("should not show the foreign key target for non-foreign keys", async () => {
-      await setup();
-
-      await userEvent.click(screen.getByText(ORDERS_TABLE.display_name));
-      const section = within(
-        await screen.findByLabelText(ORDERS_ID_FIELD.name),
-      );
-
-      expect(section.queryByText("Products → ID")).not.toBeInTheDocument();
-    });
-
-    it("should show currency settings for currency fields", async () => {
-      await setup();
-      await userEvent.click(screen.getByText(ORDERS_TABLE.display_name));
-
-      const section = within(
-        await screen.findByLabelText(ORDERS_DISCOUNT_FIELD.name),
-      );
-      const currencyInput = section.getByPlaceholderText(
-        "Select a currency type",
-      );
-      await userEvent.click(currencyInput);
-
-      const popover = within(await screen.findByRole("listbox"));
-      expect(popover.getByText("Canadian Dollar")).toBeInTheDocument();
-      expect(popover.getByText("Euro")).toBeInTheDocument();
-
-      await userEvent.clear(currencyInput);
-      await userEvent.type(currencyInput, "Dollar");
-
-      expect(popover.getByText("US Dollar")).toBeInTheDocument();
-      expect(popover.getByText("Canadian Dollar")).toBeInTheDocument();
-      expect(popover.queryByText("Euro")).not.toBeInTheDocument();
-    });
-
-    it("should not show currency settings for non-currency fields", async () => {
-      await setup();
-      await userEvent.click(screen.getByText(ORDERS_TABLE.display_name));
-
-      const section = within(
-        await screen.findByLabelText(ORDERS_ID_FIELD.name),
-      );
-      expect(section.queryByText("US Dollar")).not.toBeInTheDocument();
-    });
-  });
-
-  describe("multi schema database", () => {
-    it("should not select the first schema if there are multiple schemas", async () => {
-      await setup({ databases: [SAMPLE_DB_MULTI_SCHEMA] });
-
-      expect(screen.getByText(SAMPLE_DB_MULTI_SCHEMA.name)).toBeInTheDocument();
-      expect(
-        screen.getByText(PEOPLE_TABLE_MULTI_SCHEMA.schema),
-      ).toBeInTheDocument();
-      expect(
-        screen.getByText(REVIEWS_TABLE_MULTI_SCHEMA.schema),
-      ).toBeInTheDocument();
-      expect(
-        screen.queryByText(PEOPLE_TABLE_MULTI_SCHEMA.display_name),
-      ).not.toBeInTheDocument();
-    });
-
-    it("should allow to search for a schema", async () => {
```

Comment:
This is no longer a thing. Previously there were 3 different searches (db, schema, table), now there is one search (table).